### PR TITLE
games-fps/yamagi-quake2: version bump to 7.42

### PR DESCRIPTION
**Pull request by puleglot**

### CODE REVIEW

File: games-fps/yamagi-quake2/yamagi-quake2-7.42.ebuild [hunk 1]
```diff
@@ -0,0 +1,154 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit desktop eutils
+
+CTF_V=1.06
+ROGUE_V=2.06
+XATRIX_V=2.07
```
Comment (juippis) [inline games-fps/yamagi-quake2/yamagi-quake2-7.42.ebuild]: These looks like strings, could you mark them as ones? 
" "

File: games-fps/yamagi-quake2/yamagi-quake2-7.42.ebuild [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+SRC_URI="https://deponie.yamagi.org/quake2/quake2-${PV}.tar.xz
+	ctf? ( https://deponie.yamagi.org/quake2/quake2-ctf-${CTF_V}.tar.xz )
+	rogue? ( https://deponie.yamagi.org/quake2/quake2-rogue-${ROGUE_V}.tar.xz )
+	xatrix? ( https://deponie.yamagi.org/quake2/quake2-xatrix-${XATRIX_V}.tar.xz )"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="+client ctf dedicated openal +opengl rogue softrender xatrix"
+REQUIRED_USE="|| ( client dedicated ) client? ( || ( opengl softrender ) )"
```
Comment (juippis) [inline games-fps/yamagi-quake2/yamagi-quake2-7.42.ebuild]: Please multi-line this to make it easier to read.

File: games-fps/yamagi-quake2/yamagi-quake2-7.42.ebuild [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+		openal? ( media-libs/openal )
+		!openal? ( media-libs/libsdl2[sound] )
+		opengl? (
+			media-libs/libsdl2[opengl]
+			virtual/opengl
+		)
+	)
+"
+RDEPEND="${COMMON_DEPEND}
+	client? ( openal? ( media-libs/openal ) )
```
Comment (juippis) [inline games-fps/yamagi-quake2/yamagi-quake2-7.42.ebuild]: Doesn't this already come from inheriting `COMMON_DEPEND`?

File: games-fps/yamagi-quake2/yamagi-quake2-7.42.ebuild [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+"
+
+DEPEND="${COMMON_DEPEND}"
```
Comment (juippis) [inline games-fps/yamagi-quake2/yamagi-quake2-7.42.ebuild]: So `COMMON_DEPEND` is not needed.

File: games-fps/yamagi-quake2/yamagi-quake2-7.42.ebuild (hunk 3 above)
Comment (puleglot) [reply games-fps/yamagi-quake2/yamagi-quake2-7.42.ebuild]: It's runtime only dependency, so it shouldn't be in `COMMON_DEPEND`

**Pull request closed without merging**